src/test/java/code401challenges/linkedlist/LinkedListTest.java

```diff
@@ -0,0 +1,102 @@
+package code401challenges.linkedlist;
+
+import org.checkerframework.common.value.qual.StaticallyExecutable;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class LinkedListTest {
+    @Test
+    public void linkedListTest() {
+        LinkedList test = new LinkedList();
+        assertNull(test.head);
+    }
+
+
+
+//    2.Can properly insert into the linked list
+    @Test
+    public void prependTest() {
+        LinkedList test = new LinkedList();
+        test.prepend(7);
+        int actual = test.head.value;
+//        check if head is pointing to node with value of 7
+       assertEquals(7, actual);
+    }
+
+
+
+//    3.The head property will properly point to the first node in the linked list
+    @Test
+    public void 

```

Comment:
It's excellent to have notes on what tests to add later, but either delete the incomplete method signature here or comment it out, because it prevents me from running all your tests at once.